File: PP_02.01/DrivingSchoolWebAPI2/DrivingSchoolAPIModels/ApiModels/ClassModel.cs
namespace DrivingSchoolAPIModels
{
    public class ClassModel
    {
        public int? StudentId { get; set; }
        public TimeSpan StartTime { get; set; }
        public TimeSpan Duration { get; set; }
        public int? InstructorId {  get; set; }
        public int? InnerScheduleOfInstructorId { get; set; }
    }
}

File: PDP/DrivingSchoolAppAndroidClient/app/src/main/java/com/example/drivingschoolappandroidclient/api/models/ClassModel.kt
package com.example.drivingschoolappandroidclient.api.models

import com.example.drivingschoolappandroidclient.App
import kotlinx.serialization.Serializable
import java.sql.Date
import java.time.LocalTime

data class ClassModel(
    var studentId: Int? = null,
    var startTimeJson: String,
    var durationJson: String,
    var instructorId: Int? = null,
    var innerScheduleOfInstructorId: Int? = null
){
    var startTime: LocalTime
        get() = App.timeFromString(startTimeJson)
        set(value) {
            startTimeJson = App.timeToString(value)
        }
    var duration: LocalTime
        get() = App.timeFromString(durationJson)
        set(value) {
            durationJson = App.timeToString(value)
        }
}